lints/queuelint.py

```diff
@@ -24,7 +24,7 @@
         queues_list = item.value
 
 if queues_list is None:
-    print "Failed to find list of queues in settings file"
```

Comment:
Could you add `from __future__ import print_function` to the files that use `print()`? (Whilst Python 2 doesn't report a syntax error when using `print()` it's technically still a print statement rather than function)

Reply:
TIL!

Fixed in 33669fc1